src/main/java/org/folio/linked/data/configuration/kafka/LinkedDataTopicProperties.java

```diff
@@ -10,5 +10,6 @@
 public class LinkedDataTopicProperties {
 
   private String searchBibframeIndex;
+  private String searchBibframeAuthoritiesIndex;
```

Comment:
You can address this comment as part of this PR or as part of https://folio-org.atlassian.net/browse/MSEARCH-786.

Let us rename `searchBibframeAuthoritiesIndex` to `authoritiesSearchIndex` and rename the existing variable `searchBibframeIndex` to `workSearchIndex`

Reply:
add comment to that task